### PR TITLE
add get trade logs v1

### PR DESCRIPTION
**Pull request by onlylove97**

### CODE REVIEW

**Comment by secmask** [comment]
@onlylove97 CI failed, fix it please.

**Review by halink0803 — CHANGES_REQUESTED** [review]
Now we usdt as main quote, so, please remove all the part related to the ETH (as quote) and ETHUSDT price as well.

File: tradelogs/storage/postgres/postgres.go [hunk 1]
```diff
@@ -133,12 +133,12 @@ func (tldb *TradeLogDB) tradeLogFromDBData(r tradeLogDBData) (common.Tradelog, e
...
-		Index:             r.LogIndex,
-		Timestamp:         r.Timestamp,
-		BlockNumber:       r.BlockNumber,
-		EthAmount:         ethAmountInWei,
-		OriginalEthAmount: originalEthAmountInWei,
+		TransactionHash:    ethereum.HexToHash(r.TxHash),
+		Index:              r.LogIndex,
+		Timestamp:          r.Timestamp,
+		BlockNumber:        r.BlockNumber,
+		USDTAmount:         ethAmountInWei,
```
Comment (halink0803) [inline tradelogs/storage/postgres/postgres.go:140]: why don't you change this "ethAmountInWei" name?

**Review by halink0803 — CHANGES_REQUESTED** [review]
Please check again.

**Review by halink0803 — APPROVED** [review]

**Pull request merged**